src/ui/studio/save-creation/index.js

```diff
@@ -330,6 +349,35 @@ const UploadForm = ({ opencast, uploadState, recordings, handleUpload }) => {
           onChange={handleInputChange}
         />
       </FormField>
+      <div sx={{
+        display : displayDescription,
+        marginBottom : '30px'
+      }}>
```

Comment:
Instead of hiding the field via `display: none`, please do not add it to the DOM at all. For example via `{ upload?.enableDescription && <Input ...> }` or something like that.